src/translations/pages/assistant.ts

```diff
@@ -407,3 +404,28 @@ export const Assistant = {
     },
   },
 };
+
+export const Assistant = orgSpecificTranslations(AssistantInternal, {
+  nfk: {
+    search: {
+      lineFilter: {
+        example: _(
+          'Eksempel: 100, 200, 300',
+          'Example: 100, 200, 300',
+          'Eksempel: 100, 200, 300',
+        ),
+      },
+    },
+  },
+  fram: {
+    search: {
+      lineFilter: {
+        example: _(
+          'Eksempel: 901, 800',
+          'Example: 901, 800',
+          'Eksempel: 901, 800',
```

Comment:
When thinking about it, these line examples might not make much sense for FRAM. These were just added as an example when I added the feature. Not sure if FRAM has a line 800 at all 🤔

Reply:
I thought about that also, just accepted that you had thought of this as lazy as I was 😅

Reply:
Checking with Christoffer. 701 is also probably relevant for Molde